index.js

```diff
@@ -427,3 +427,19 @@ module.exports.sync = (command, args, options) => {
 };
 
 module.exports.shellSync = (command, options) => handleShell(execa.sync, command, options);
+
+module.exports.fork = (filePath, args, options) => {
+	const stdioOption = stdio.fork(options);
+	delete options.stdin;
+	delete options.stdout;
+	delete options.stderr;
+
+	return execa(filePath, args, {
+		...options,
+		code: 0,
```

Comment:
There is no `code` option in execa.

Reply:
This was a test I commit by mistake, it's removed on my local directory.